mkdocs.yml

```diff
@@ -0,0 +1,124 @@
+# Project information
+site_name: 'React UI docs'
+site_description: 'React UI docs.'
+
+# Repository
+repo_url: 'https://github.com/react-ui-org/react-ui'
+repo_name: 'react-ui'
+edit_uri: 'blob/master/src'
+
+docs_dir: 'src'
+
+markdown_extensions:
```

Comment:
If I try to edit code snippet and I press key "s", search bar is opened instead of editing code snippet. (In all browsers):

![image](https://user-images.githubusercontent.com/10213870/209099766-0441f5c4-639c-4828-91d7-c271ba4f9c46.png)

Reply:
fixed